pkgs/tools/system/swaysettings/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
+  vala,
+  pkg-config,
+  gettext,
+  ninja,
+  python3Full,
+  wrapGAppsHook,
+  appstream-glib,
+  desktop-file-utils,
+  gsettings-desktop-schemas,
+  glib,
+  gtk3,
+  libhandy,
+  json-glib,
+  libgee,
+  pantheon,
+  libxml2,
+  accountsservice,
+  gobject-introspection,
+  dbus
+}:
+with pkgs;
```

Comment:
```suggestion
```

As said before, `pkgs` are not put under the whole scope.

Reply:
Ok corrected